labop/__init__.py

```diff
@@ -276,7 +276,7 @@ def show_libraries():
     for lib in labop.loaded_libraries.keys():
         show_library(lib)
 
-def get_primitive(doc: sbol3.Document, name: str):
+def get_primitive(doc: sbol3.Document, name: str, copy_to_doc: bool = True):
```

Comment:
I can anticipate the `copy_to_doc` flag might be useful in some cases.  I'm curious, what was the use case that caused you to put it in?